### DISCUSSION TITLE
Add Magazine Publication Information for Manga Series

### DISCUSSION THREAD

**Discussion by minahoku** [gotson/komga#1714] [Ideas]
As a Japanese user, I'd like to request a feature to include magazine information for manga series. In Japan, manga is typically published in magazines before being compiled into volumes. Therefore, we need a place to specify the publication magazine(s) for a series.

Since manga series can sometimes switch magazines, it would be helpful if we could enter multiple magazine entries for a single series.

**Comment by gotson** [comment]
you can use tags
